src/index.tsx

```diff
@@ -147,6 +147,7 @@ if (BUGSNAG_API_KEY) {
     "deloitte",
```

Comment:
should deloitte be `env/deloitte?`

Reply:
The `env/` pattern is something I only implemented in the last round of environments. Eventually it would be good to align them all to following this pattern, but that is OOS of this PR.

There will be some code changes, and possibly some nuance (eg. in how we map it to existing bugsnag data/etc) if/when we decide to do this.